nextflow.config

```diff
@@ -10,7 +10,7 @@ manifest {
     homePage        = 'https://github.com/UMCUGenetics/GIABEval'
     description     = """UMCU Genetics Genome-in-a-bottle evaluation workflow for benchmarking"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=23.04.0'
+    nextflowVersion = '>=23.10.0'
```

Comment:
why did you choose version 23.10.0 here and version 23.10.1 in the install script? It will work, but for validation purposes I would equate these two numbers.